xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2519 @@
+                <para>
+                  <parameter>load_threshold_powersave</parameter>: Threshold for load to switch to
+                  powersave mode. Default value is <literal>0.4</literal>.
+                </para>
+              </listitem>
+              <listitem>
+                <para>
+                  <parameter>she_powersave</parameter>: Value to set SHE to powersave mode. Default
+                  value is <literal>2</literal>.
+                </para>
+              </listitem>
+              <listitem>
+                <para>
+                  <parameter>she_normal</parameter>: Value to set SHE to normal mode. Default value
+                  is <literal>1</literal>.
+                </para>
+              </listitem>
+            </itemizedlist>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+    </sect2>
+  </sect1>
+  <sect1 xml:id="sec-tuning-tuned-more">
+    <title>More information</title>
+
+    <para>
+      For more information, refer to the following resources:
+    </para>
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          The official website of the &tunedapp; project:
+          <link xlink:href="https://tuned-project.org/"></link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Source code repository on GitHub:
+          <link
+          xlink:href="https://github.com/redhat-performance/tuned"></link>. You may
+          find the following resources to be particularly helpful:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              Configuration files for several profiles are available at
+              <link
+              xlink:href="https://github.com/redhat-performance/tuned/tree/&tunedver;/profiles"></link>.
+              These configuration examples can provide good starting point for creating a custom
+              profile that is not available as part of the <package>tuned</package> package for
+              &sles;.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              If you are interested in the implementation of the plug-ins that monitors and
+              dynamically tunes the system based on values provided in the profiles, the source
+              code is available at
+              <link xlink:href="https://github.com/redhat-performance/tuned/tree/&tunedver;/tuned/plugins"></link>.
+            </para>
+          </listitem>
+        </itemizedlist>
+      </listitem>
+    </itemizedlist>
+
+    <tip>
+      <title>Refer to appropriate version of &tunedapp; source code</title>
+      <para>
+        Check for the version of &tuned; in your &sles; system by running the <command>tuned
+        --version</command> command. On the GitHub repository, select the corresponding branch or
+        tag for inspecting the source code of the specific &tuned; version installed on your
+        system.
+      </para>
+    </tip>
```

Comment:
That's certainly a good tip, but I think there is a purpose missing: why should a customer do that? When do they have to do it? Maybe to help support? To review the changelog? To find security fixes?

For two latter use cases it would be better to use `rpm -q --changelog tuned` to view the changelog including any CVS security fixes. Looking only on GitHub can be misleading as patches can be applied in the SUSE RPM package, but not yet in GitHub.

Reply:
TuneD profiles and use of plug-ins are highly customizable, to suit many use cases. However, the standard stuff installed with the official package is very limited. I have given links to several GitHub resources earlier in the article, so that customers can get ideas and inspiraton for what's possible, and they can experiment a bit. 
However, the version of TuneD shipped with SLES15SP6 is v2.10.0, which is 5-6 years behind the latest version. If a customer, by mistake, takes inspiration from the latest branches or tags in the GitHub repository, instead of the branch/tag that corresponds to the installed version of TuneD, then that can be disastrous. That's because later versions of TuneD has features or capabilities that makes certain customizations of profiles and plug-ins technically valid, while the older versions such as v2.10.0 doesn't have those fancy bells and whistles. I wrote the tip so that customers don't make this mistake of taking customization ideas from the latest branches/tags, and try to make them work with older installed version of TuneD.
I hope I was able to explain my intent. I would be happy to hop on a call with you and discuss this further, if required.